src/scheduler.rs

```diff
@@ -46,8 +46,8 @@ impl Scheduler {
         }
     }
 
-    fn init_seed(parameters: &mut Parameters, current: &Card, now: DateTime<Utc>) {
-        let time = now.timestamp_millis();
+    fn init_seed(parameters: &mut Parameters, current: &Card) {
+        let time = Utc::now().timestamp_millis();
         let reps = current.reps;
         let mul = current.difficulty * current.stability;
         parameters.seed = Seed::new(format!("{}_{}_{}", time, reps, mul));
```

Comment:
```rs
parameters.seed = Seed::new(format!("{time}_{reps}_{mul}"));
```

Reply:
Wait. I feel confused. Maybe it's a problem from the source: ts-fsrs.

https://github.com/open-spaced-repetition/ts-fsrs/issues/131